README.md

```diff
@@ -162,7 +162,174 @@ useradd --system --user-group --create-home myapp
 
 That should do it.
 
-###
+## Truecolor Over SSH With Wish
+
+### Noteworthy environment variables (for debugging)
+
+`TERM` - provides information about your terminal emulator's capabilities.
+
+`COLORTERM` - provides information about your terminal emulator's color
+capabilities. Used primarily to specify if your emulator has `truecolor`
+support.
+
+`NO_COLOR` - turns colors on and off. `NO_COLOR=1` for non-colored text
+outputs, `NO_COLOR=0` for colored text outputs.
+
+`CLICOLOR` - turns colors on and off. `CLICOLOR=1` for colored text outputs,
+`CLICOLOR=0` for non-colored text outputs.
+
+`CLICOLOR_FORCE` - overrides `CLICOLOR`.
```

Comment:
I believe none of these are sent over SSH by default.

```bash
ssh -v localhost
```

look for `setting env`.


I think the ones sent by default are `TERM`, `LANG`, and `LC_*`